jobconfig.toml

```diff
@@ -109,49 +111,49 @@
 [COOK]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
+"Spawn Positions" = 2
 "Total Positions" = 2
 
 [CORRECTIONS_OFFICER]
 "Playtime Requirements" = 450
 "Required Account Age" = 7
-"Spawn Positions" = 1
-"Total Positions" = 1
+"Spawn Positions" = 2
+"Total Positions" = 2
```

Comment:
.
I can't tell how many icebox has easily because I believe they use the automapper

Reply:
Looked at this and reduced spawn positions from 2 to 1 in next commit. If confirmed that 2 is a value okay with maps like Icebox, then will raise it again in a new PR.

Reply:
This wasn't actually reduced to 1